src/components/DocumentationTopic.vue

```diff
@@ -18,16 +18,12 @@
         <div v-if="sampleCodeDownload">
           <DownloadButton class="sample-download" :action="sampleCodeDownload.action" />
         </div>
-        <Availability v-if="platforms" :platforms="platforms" />
+        <Availability
+          v-if="!hideSummary"
```

Comment:
```suggestion
          v-if="platforms.length || technologies.length"
```

Reply:
Or `platforms.length || (technologies || [].length)` actually for the 2nd bit if you keep `technologies` as an optional prop.